tools/libboard/board_common/board_common.c

```diff
@@ -26,7 +26,7 @@
 
 #ifdef HAVE_CONFIG_H
 #include <config.h>
-#endif // HAVE_CONFIG_H
+#endif // HAVE_CONFIG_Hrm
```

Comment:
Remove the "rm" from the end of "HAVE_CONFIG_H"

Reply:
fixed remove rm